source/game.screen.stationmap.bmx

```diff
@@ -1397,6 +1398,7 @@ endrem
 
 						Local finance:TPlayerFinance = GetPlayerFinance(TScreenHandler_StationMap.currentSubRoom.owner)
 						canAfford = (Not finance Or finance.canAfford(selectedStation.GetPrice()))
+						print "tada"
```

Comment:
Tadaaaa!

Reply:
Ja, meine typischen "Komme ich hier im Code vorbei"-Debugmeldungen sind tada und tidum.

Reply:
Das macht man vielleicht in LE so - in Chemnitz wird

print "schluesselwort 1"
print "schluesselwort 2"
print "schluesselwort 3"
print "schluesselwort 4"
... genutzt, da erkennt man wenigstens an welcher Stelle das ist. ... ich mag Debugbuilds nicht so, die "ruckeln" ;-)